front/app/modules/commercial/insights/admin/containers/Insights/Edit/Edit.test.tsx

```diff
@@ -1,138 +1,11 @@
 import React from 'react';
-import { render, screen, fireEvent, act } from 'utils/testUtils/rtl';
-import * as service from 'modules/commercial/insights/services/insightsCategories';
-import clHistory from 'utils/cl-router/history';
+import { render, screen } from 'utils/testUtils/rtl';
 
 import InsightsEdit from './';
 
-let mockData = [
-  {
-    id: '1aa8a788-3aee-4ada-a581-6d934e49784b',
-    type: 'category',
-    attributes: {
-      name: 'Test',
-    },
-  },
-  {
-    id: '4b429681-1744-456f-8550-e89a2c2c74b2',
-    type: 'category',
-    attributes: {
-      name: 'Test 2',
-    },
-  },
-];
-
-jest.mock('utils/cl-intl');
-
-jest.mock('modules/commercial/insights/services/insightsCategories', () => ({
-  addInsightsCategory: jest.fn(),
-  deleteInsightsCategories: jest.fn(),
-  deleteInsightsCategory: jest.fn(),
-}));
-
-jest.mock('modules/commercial/insights/hooks/useInsightsCategories', () => {
-  return jest.fn(() => mockData);
-});
-
-jest.mock('hooks/useLocale', () => jest.fn(() => 'en'));
-
-const viewId = '1';
-
-const mockLocationData = { pathname: '', query: {} };
-
-jest.mock('react-router', () => {
-  return {
-    withRouter: (Component) => {
-      return (props) => {
-        return (
-          <Component
-            {...props}
-            params={{ viewId }}
-            location={mockLocationData}
-          />
-        );
-      };
-    },
-    Link: () => 'Link',
-  };
-});
-
-window.confirm = jest.fn(() => true);
-
 describe('Insights Edit', () => {
   it('renders Edit screen', () => {
     render(<InsightsEdit />);
     expect(screen.getByTestId('insightsEdit')).toBeInTheDocument();
   });
-  describe('Categories', () => {
```

Comment:
Nice-to-have : check all the components rendered are present, not only the container

Reply:
The rest of the components are tested in the other tests - to check if they render, we need to repeat all the mocks in this test as well and I don't see a real need for it :)